qa/pull-tester/rpc-tests.py

```diff
@@ -266,7 +268,16 @@ def main():
         subprocess.check_call((config["environment"]["SRCDIR"] + '/qa/rpc-tests/' + test_list[0]).split() + ['-h'])
         sys.exit(0)
 
-    run_tests(test_list, config["environment"]["SRCDIR"], config["environment"]["BUILDDIR"], config["environment"]["EXEEXT"], args.jobs, args.coverage, passon_args)
+    if((args.rpcgroup == -1 and args.mahines != -1) or (args.rpcgroup ==-1 and args.rpcmachines != -1)):
```

Comment:
```suggestion
    if (args.rpcgroup == -1) != (args.machines == -1):
```
This corrects a typo as well as being simpler.